gammapy/analysis/core.py

```diff
@@ -270,8 +270,8 @@ def write_datasets(self, overwrite=True, write_covariance=True):
 
         Parameters
         ----------
-        overwrite : bool
-            overwrite datasets FITS files
+        overwrite : bool, optional
+            Overwrite existing file. Default is False.
```

Comment:
The default is True here.

Reply:
Probably we should make it consistent as well, however for the HLI it might have been a deliberate decision.

Reply:
Happy to change it to 'False' unless there is a reason we can think why the default should be True